### PR TITLE
Support storage of Seurat cell identities

### PR DESCRIPTION
**Pull request by aaronwolen**
As documented in `SCDataset`:

> Cell identities in the [`SeuratObject::Seurat`] are maintained by creating an `active_ident` attribute in `obs` that stores the factor levels as a character vector.

### CODE REVIEW

**Comment by aaronwolen** [comment]
CC @dnadave @kaitlin-procogia @augustine-procogia

File: R/SCDataset.R [hunk 1]
```diff
@@ -68,6 +74,15 @@ SCDataset <- R6::R6Class(
     from_seurat = function(object) {
       stopifnot(inherits(object, "Seurat"))
 
+      identities <- SeuratObject::Idents(object)
```
Comment (johnkerl) [inline R/SCDataset.R:77]: i would prefer calling these `idents` (even though that sounds weird) in order to harmonize with the seurat names here

**Review by johnkerl — APPROVED** [review]
🚢 with nit :)

File: R/SCDataset.R (hunk 1 above)
Comment (aaronwolen) [reply R/SCDataset.R:77]: You've won me over.

Comment (aaronwolen) [reply R/SCDataset.R]: https://github.com/TileDB-Inc/tiledbsc/pull/56/commits/ca1ac083de9b360f4483fee76260c05f33fac117

**Pull request merged**